### PR TITLE
Fix XML code blocks 

### PR DESCRIPTION
**Pull request by cloudn9**
Continuing list from #9744

### CODE REVIEW

File: docs/visual-basic/misc/bc36558.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ End Namespace
 Alternatively, you could allow `ExtensionAttribute` to apply to all program elements by using the `All` member of `AttributeTargets`.
 
 ```xml
-<AttributeUsage(AttributeTargets.All)>
+<AttributeUsage(AttributeTargets.All)> _
```
Comment (gewarren) [inline docs/visual-basic/misc/bc36558.md]:
```suggestion
<AttributeUsage(AttributeTargets.All)>
```

Only the original example on line 31 was a run-on line.

**Review by gewarren — CHANGES_REQUESTED** [review]
One change to make.

File: docs/framework/data/adonet/connection-strings-and-configuration-files.md [hunk 2]
```diff
@@ -119,9 +119,9 @@ Embedding connection strings in your application's code can lead to security vul
 <configProtectedData defaultProvider="RsaProtectedConfigurationProvider">  
   <providers>  
     <add name="RsaProtectedConfigurationProvider"
-      type="System.Configuration.RsaProtectedConfigurationProvider, ... />  
+      type="System.Configuration.RsaProtectedConfigurationProvider />  
     <add name="DataProtectionConfigurationProvider"
-      type="System.Configuration.DpapiProtectedConfigurationProvider, ... />  
+      type="System.Configuration.DpapiProtectedConfigurationProvider />
```
Comment (Thraka) [inline docs/framework/data/adonet/connection-strings-and-configuration-files.md]: Missing end qoute
```suggestion
      type="System.Configuration.DpapiProtectedConfigurationProvider" />  
```

**Review by Thraka — APPROVED** [review]

**Comment by Thraka** [comment]
@gewarren Since you have a pending review request, I'll let you finish approval+merge. 👍

**Review by gewarren — APPROVED** [review]

**Comment by cloudn9** [comment]
Oh thank you both @gewarren @Thraka for correcting those.

**Pull request merged**